# This is an example configuration that loads the "Kitchen" scene from the
# local MoveIt! warehouse and benchmarks the "manipulator" group in the Pick1
# query using the Start1 initial state (all pre-stored in the local warehouse)

# Three different planners from OMPL are run a total of 50 times each, with a
# maximum of 10 seconds per run.  Output is stored in the /tmp/moveit_benchmarks directory.

benchmark_config:
    warehouse:
        host: 127.0.0.1
        port: 33829
        scene_name: scene_1     # Required
    parameters:
        name: scene_1_query_1
        runs: 150
        group: arm       # Required
        timeout: 10.0
        output_directory:  /home/charlene/catkin_ws/src/crane_plus_v2_motion_planning/crane_plus_control/results/benchmarks/logs/
        queries: [query_1, query_2, query_3]
        start_states: vertical
    planners:
        - plugin: ompl_interface/OMPLPlanner
          planners:
            - RRTConnectkConfigDefault
            - BiTRRTkConfigDefault
            - BKPIECEkConfigDefault
            - KPIECEkConfigDefault


